src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -205,6 +206,7 @@ public void updateS3UploadBannerView(String banner) {
 	
 	public void queryForUploadDestination() {
 		enableMultipleFileUploads();
```

Comment:
Please add a few tests around this method, to verify that the storage location id has the expected value when the first UploadDestination is of different types (if is empty or if no parent then storage location should be null, if S3 or External then it should be whatever was set in the UploadDestination)